manage.py

```diff
@@ -36,6 +37,26 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.command
+def switch(system):
```

Comment:
See the option below. Rename to module and pass name and on/off as parameters

Reply:
ok, on it

Reply:
Implemented, please review